src/index.js

```diff
@@ -298,30 +329,33 @@ const createGlobalHeader = ({ headerTitle, button1Label, button1OnClick, button2
     }, headerElement)
 }
 
-const viewContact = () => {
+const viewContact = (person) => {
     handleFooterElementOnClick("view-contact")
     removeElementsOnTabChange("view-contact")
     createGlobalHeader({
         headerTitle: "",
         button1Label: INNER_TEXTS.backButtonElement,
         button1OnClick: renderContacts,
         button2Label: INNER_TEXTS.editButtonElement,
-        button2OnClick: console.log("This will edit the contact"),
+        button2OnClick: () => console.log("This will edit the contact"),
         parentElement: viewContactElement
     })
 
     // MAIN
     const mainElement = createElement("main", undefined, viewContactElement)
 
-    const photoElement = createElement("div", {
-        id: ELEMENT_IDS.photo
-    }, mainElement)
-
     const contactNameElement = createElement("p", {
         id: ELEMENT_IDS.contactName,
-        innerText: "Name will go here"
+        innerText: `${person.firstName} ${person.lastName}`,
     }, mainElement)
 
+    const photoElement = createElement("div", {
+        id: ELEMENT_IDS.contactPhoto,
+        innerText: contactNameElement.innerText.slice(0, 1)
```

Comment:
353 - 357
- 354 - is this necessary?
- If no, then instead 357 you could simply use 3rd param in createElement to append elements to parent